README.md

```diff
@@ -8,7 +8,7 @@
 
 ## Quick-Start a Stacks.js Project
```

Comment:
Could we add an intro/overview paragraph to describe this repo's intent or objective? Explaining that first and getting into the usage steps next would be helpful.

IMHO, this project aims to help quickly bootstrap frontend applications with Stacks.js on top of the desired JS framework as the foundation.